service-deployment/bootstrap/rest-server/rest-server.yaml.template

```diff
@@ -43,7 +47,7 @@ spec:
         - name: JWT_SECRET
           value: {{ clusterinfo['restserverinfo']['jwt_secret'] }}
         - name: LOWDB_FILE
-          value: {{ clusterinfo['restserverinfo']['lowdb_file'] }}
+          value: /var/databases/{{ clusterinfo['restserverinfo']['lowdb_file'] }}
```

Comment:
Configurable?

Reply:
Do not need to make it configurable inside container.